libcontainer/configs/config.go

```diff
@@ -171,6 +171,9 @@ type Config struct {
 	// A default action to be taken if no rules match is also given.
 	Seccomp *Seccomp `json:"seccomp"`
 
+	// NoNewPrivileges controls whether processes in the container can gain additional privileges.
+	NoNewPrivileges bool `json:"nonewprivileges"`
```

Comment:
Not a problem in this PR.
For json property names, we have `nonewprivileges` and `noNewPrivileges` and `no_new_privileges` in code, looks like we don't have code style for that yet, should we change them for consistency.